File: Gemini Advanced/Python/2-3.py
def file_read_write():
    try:
        with open("input.txt", "r") as infile, open("output.txt", "w") as outfile:
            content = infile.read()
            outfile.write(content)
        return "File content copied successfully!"
    except FileNotFoundError:
        return "Error: Input file not found."
    except IOError as e:
        return f"An error occurred: {e}"

print(file_read_write())

File: gpt4/C++/2-3.c
#include <iostream>
#include <fstream>
using namespace std;

int main() {
    ifstream inFile("input.txt");
    ofstream outFile("output.txt");
    if (!inFile.is_open() || !outFile.is_open()) {
        cout << "Error opening files!" << endl;
        return 1;
    }
    string line;
    while (getline(inFile, line)) {
        outFile << line << endl;
    }
    inFile.close();
    outFile.close();
    cout << "File has been copied." << endl;
    return 0;
}


